.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- Complete this Pull Request template. -->
+
+<!-- Significant PRs should address an existing Issue. Choose one: -->
+
+These changes partially address #xxxx
+These changes close #xxxx
+This is a small change with no associated Issue.
+
+<!-- The following requirements must be satisfied (with "[x]"). -->
+<!-- Mark the PR as a Draft if all requirements are not yet satisfied. -->
+
+**Requirements check-list**
+- [ ] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
+- [ ] Contains logically grouped changes (else tidy your branch by rebase).
+- [ ] Does not contain off-topic changes (use other PRs for other changes).
+<!-- choose one: -->
+- [ ] Appropriate tests are included (unit and/or functional).
+- [ ] Already covered by existing tests.
+- [ ] Does not need tests (why?).
+<!-- choose one: -->
+- [ ] Appropriate change log entry included.
+- [ ] No change log entry required (why? e.g. invisible to users).
+<!-- choose one: -->
+- [ ] I have opened a documentation PR at cylc/cylc-doc/pull/XXXX.
```

Comment:
(I guess we don't really have any UI documentation yet - but can worry about that later)